app/model/des/CustomerInformation.scala

```diff
@@ -16,27 +16,40 @@
 
 package model.des
 
-import play.api.libs.json.{JsResult, JsValue, Reads}
+import play.api.libs.json._
 
-/**
- * Case class representing a small subset of the DES 1363 API response
- * containing customer data for a VRN.  Currently, we only extract the Welsh Indicator,
- * but more data can be added in future
- */
-final case class CustomerInformation(welshIndicator: Option[WelshIndicator])
+final case class CustomerInformation(approvedInformation: Option[ApprovedInformation]) {
+  def unWrap(vrn: String): ApprovedInformation = {
+    approvedInformation match {
+      case None       => throw new RuntimeException(s"""No Customer data for VRN: ${vrn}""")
```

Comment:
hmm, maybe we can extend model so there would be:
a)  `CustomerInformationRaw` - a case class with options returned by from connector
b) `CustomerInformation` - a case class without options, projection from former one, returned by service. All these exceptions would be thrown from service.

It's easier to debug if an exception is thrown, and there is one place (service) which can cause exceptions.